brewery-app/src/components/MapContainer.js

```diff
@@ -12,45 +13,85 @@ const StyledMap = Styled.div`
   border-radius: 10px;
 `
 
-const AnyReactComponent = ({ text }) => <div>{text}</div>;
 const ResultPin = ({brewery}) => <MapModal brewery={brewery} />;
  
 const MapContainer = (props) => {
+  console.log(props.searchResults)
+  const [DefaultMapData, setDefaultMapData] = useState({
+    lat: 41.559,
+    lng: -90.483,
+    zoom: 11
+  })
+  
+  const calcCenter = (set, type) => {
+    console.log(set)
+    if(set.length > 0) {
+      return set.reduce((a,b) => parseFloat(a) + parseFloat(b), 0) / set.length;
+    } else {
+      if(type === "lat") {
+        return DefaultMapData.lat;
+      } else if(type === "lng") {
+        return DefaultMapData.lng;
+      }
+    }
+  }
+
+  let lats=[];
```

Comment:
These arrays can be saved using useState, then they won't be floating outside of a component method. 
You could also include it with the default map data in one bigger state called MapData instead of DefaultMapData